plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtStoragePoolDef.java

```diff
@@ -75,6 +80,15 @@ public LibvirtStoragePoolDef(PoolType type, String poolName, String uuid, String
         _targetPath = targetPath;
     }
 
+    public LibvirtStoragePoolDef(PoolType type, String poolName, String uuid, String host, String dir, String targetPath, List<String> nfsopts) {
+        this(type, poolName, uuid, host, dir, targetPath);
+        if (nfsopts != null) {
```

Comment:
Please use CollectionUtils.isNotEmpty() and in other places as well

Reply:
Done